examples/synergetic.py

```diff
@@ -0,0 +1,60 @@
+import os
+import random
+import json
+
+from fetchai.ledger.api import LedgerApi
+from fetchai.ledger.crypto import Entity, Address
+from fetchai.ledger.contract import SynergeticContract
+
+CONTRACT_TEXT = """
+@problem
```

Comment:
if you are up to it I would comment this fairly liberally since it's not clear what is going on

Reply:
I plan on updating this with a much better and representative one in the future. I will ensure that is commented heavily